bibtasklets/bst_fermilab.py

```diff
@@ -225,6 +232,20 @@ def bst_fermilab():
     write_message('\\rm fermilab-reports-preprints.html')
     write_message('cp %s .' % filename)
 
+    if len(ERRORS) > 0:
+        send_notification_email(ERRORS)
 
+def send_notification_email(ERRORS):
+    """Notify on errors by mail."""
+
+    msg_html = """<p>Message from BibTasklet bst_fermilab:</p>
+<p>Problems have occurred in %d author identifier(s):'</p>
+
+""" % (len(ERRORS))
+    for e in ERRORS:
```

Comment:
I don't understand this, ERRORS is a list of strings.
what is join going to do on a string?